cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,120 @@
 /// <reference types='cypress' />
 
+const { generateWorker } = require('../support/workerGenerate');
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  const { firstName, lastName, email, age, salary, department } = generateWorker();
+
+  it('should contain pagination on the page', () => {
```

Comment:
you do not check the pagination here. You need to add workers to make workers amount equal to 6. Than Change rows count to 5. Click the button next -> assert that page 2 is displayed. Click button previous -> assert that page 1 is displayed. Also in this test you can check the rows count. When you change row count to 5 -> assert that table contain 5 rows.

Reply:
I have fixed this test.